src/java/org/jdbcdslog/CallableStatementLoggingHandler.java

```diff
@@ -0,0 +1,68 @@
+package org.jdbcdslog;
```

Comment:
Not sure how or if it's worth keeping the original tests with this fork as well. Something to ponder.